render/filters.go

```diff
@@ -73,25 +73,24 @@ func ComposeFilterFuncs(fs ...FilterFunc) FilterFunc {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc   FilterFunc
-	filterPseudo bool
```

Comment:
Okay so I put this together and its a much bigger change.  I'm not super keen on the changing this, but have a look yourself and let me know:
[diff.patch.zip](https://github.com/weaveworks/scope/files/316907/diff.patch.zip)